komga-webui/src/locales/en.json

```diff
@@ -292,7 +292,7 @@
       "field_isbn": "ISBN",
       "field_isbn_error": "Must be a valid ISBN 13",
       "field_number": "Number",
-      "field_number_sort": "Sort Number",
+      "field_number_sort": "Sorting Number",
```

Comment:
Not 100% sure about this

Reply:
It's the number used for sorting, so it's called a sort number. Same as plex which has a "sort title".